### PR TITLE
Sync doc on Push feature from `push-feature-docs-alignment` branch

### PR DESCRIPTION
**Pull request by Inokinoki**
## Type of Change

- [ ] 📚 Examples / docs / tutorials / dependencies update
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [ ] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Checklist

- [ ] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CODE_OF_CONDUCT.md) document.
- [ ] I've read the [`CONTRIBUTING.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CONTRIBUTING.md) guide.
- [ ] I've updated the code style using `make codestyle`.
- [ ] I've written tests for all new methods and classes that I created.
- [ ] I've written the docstring in Google format for all the methods and classes that I used.

### CODE REVIEW

**Comment by Inokinoki** [comment]
This branch was not merged into main from #1303

File: python-client/docs/push-feature.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+"button": "Save Slice"
+```
+
+## Perturbation 
+
+Title| Description
+-- | --
+Description| Tag features that make the prediction change after a small variation
+Requirement| Applicable only to numerical features and textual feature
+Triggering event |  When we switch examples and the prediction changes after: 3 MAD (Median absolute deviation) variation or for textual feature, a small variation such as uppercase or punctuation transformation
```
Comment (rabah-khalek) [inline python-client/docs/push-feature.md:125]: This is outdated, we don't use MAD anymore

File: python-client/docs/push-feature.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+Title| Description
+-- | --
+Description|  Tag features that contribute a lot to the prediction
+Requirement| NA
+Triggering event | When we switch examples and the first or the first two most contributing shapley’s value are really high compared to the rest
+Test description |  Check if the chi 2 / Kolmogorov-Smirnov correlation statistical test between the feature and the prediction has a p-value above α=0.05
+
+```
+"push_title": f"{str(feature)}=={str(value)} contributes a lot to the prediction",
```
Comment (rabah-khalek) [inline python-client/docs/push-feature.md:76]: There's also a different `push_title` in case of a text feature

**Comment by Inokinoki** [comment]
@rabah-khalek I think I do not have enough contexts now on modifying these docs for push feature. Should we temporarily drop this PR or improve it later ?

**Comment by rabah-khalek** [comment]
outdated.

**Pull request closed without merging**